checker/src/main/java/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -166,6 +171,18 @@ private boolean containsSameByName(
         return false;
     }
 
+    @Override
+    protected void checkConstructorResult(
+            AnnotatedExecutableType constructorType, ExecutableElement constructorElement) {
+        // Nothing to check
```

Comment:
Shouldn't `@Nullable` be forbidden as constructor result? Can you add a test for this?